system/prometheus-infra/templates/_prometheus-infra.yaml.tpl

```diff
@@ -1,3 +1,116 @@
+- job_name: 'prometheus-infra-vrops'
+  scheme: https
+  scrape_interval: {{ .Values.collector.scrapeInterval }}
+  scrape_timeout: {{ .Values.collector.scrapeTimeout }}
+
+  honor_labels: true
+  metrics_path: '/federate'
+
+  params:
+    'match[]':
+      - '{job="vrops-exporter",__name__!~"^vrops_virtualmachine_.*"}'
+      - '{job="vrops-exporter",__name__=~"^vrops_virtualmachine_.*", vccluster=~".*management.*"}'
+      - '{job="vrops-exporter",__name__=~"vrops_api_response"}'
+      - '{__name__=~"vrops_inventory_.+"}'
+
+  relabel_configs:
+    - action: replace
+      source_labels: [__address__]
+      target_label: region
+      regex: prometheus-infra-collector.(.+).cloud.sap
+      replacement: $1
+
+  metric_relabel_configs:
+    - source_labels: [__name__, prometheus]
+      regex: '^up;(.+)'
+      replacement: '$1'
+      target_label: prometheus_source
+      action: replace
+    - source_labels: [__name__, ifIndex, server_id]
+      regex: '^snmp_[a-z0-9]*_if.+;(.+);(.+)'
+      replacement: '$1@$2'
+      target_label: uniqueident
+      action: replace
+    - source_labels: [__name__, instance]
+      regex: '^probe_success;(.+)'
+      replacement: '$1'
+      target_label: target
+      action: replace
```

Comment:
These relabelings are specific to `snmp` and `cloud prober` metrics and can be removed for this job.